mlir/test/Conversion/VectorToLLVM/vector-to-llvm.mlir

```diff
@@ -2564,3 +2564,39 @@ func.func @vector_deinterleave_1d_scalable(%a: vector<[4]xi32>) -> (vector<[2]xi
     %0, %1 = vector.deinterleave %a : vector<[4]xi32> -> vector<[2]xi32>
     return %0, %1 : vector<[2]xi32>, vector<[2]xi32>
 }
+
```

Comment:
The unrolling should be tested separately from the LLVM conversion. See how this is done for interleave here: https://github.com/llvm/llvm-project/blob/main/mlir/test/Dialect/Vector/vector-interleave-lowering-transforms.mlir